lib/blemulator/example/example_peripheral.dart

```diff
@@ -17,17 +14,9 @@ class SensorTag extends SimulatedPeripheral {
             id: id,
             advertisementInterval: Duration(milliseconds: 800),
             services: [
-              SimulatedService(
+              TemperatureService(
```

Comment:
it makes the sample more difficult to read for a developer that sees it the first time. maybe we should add more sophisticated device as second device?